v2/src/log_sink_etw.c

```diff
@@ -297,6 +297,10 @@ static void internal_emit_self_described_event_va(const char* event_name, uint16
                     /* Codes_SRS_LOG_SINK_ETW_01_063: [ If the property type is LOG_CONTEXT_PROPERTY_TYPE_ascii_char_ptr, a byte with the value TlgInANSISTRING shall be added in the metadata. ]*/
                     *pos = TlgInANSISTRING;
                     break;
+                case LOG_CONTEXT_PROPERTY_TYPE_bool:
+                    /* Codes_SRS_LOG_SINK_ETW_07_001: [ If the property type is LOG_CONTEXT_PROPERTY_TYPE_bool, a byte with the value TlgInBOOL32 shall be added in the metadata. ]*/
+                    *pos = TlgInBOOL32;
```

Comment:
>BOOL32 [](http://example.com/codeflow?start=32&length=6)

We have to make sure that we give the value as pointer to a 32 bit variable.
As is right now, it looks like we give it a bool, which will be an int, which might or might not have 32 bit.

Reply:
done